server/plpgsql/json.go

```diff
@@ -130,6 +130,47 @@ type plpgSQL_stmt_return struct {
 	LineNumber int32 `json:"lineno"`
 }
 
+// plpgSQL_stmt_while exists to match the expected JSON format.
+type plpgSQL_stmt_while struct {
+	Condition  cond        `json:"cond"`
+	Body       []statement `json:"body"`
+	LineNumber int32       `json:"lineno"`
+}
+
+// Convert converts the JSON statement into its output form.
+func (stmt *plpgSQL_stmt_while) Convert() (block Block, err error) {
```

Comment:
The convert functions are at the bottom of the file since we've got a lot of structs, and most of them don't have `Convert`, so it makes it easier to visually look through all the available structs without the functions in the way.